tv_apps/smart-home/js/card_picker.js

```diff
@@ -103,34 +104,36 @@
       this.gridView.scrollTo(0, scrollY);
     },
 
-    _refreshCardButtons: function(folderList, cardList, options) {
-      this.appButtons = [];
+    _refreshCardElements: function(folderList, cardList, options) {
+      this.appCardElems = [];
       this.gridView.innerHTML = '';
 
       var that = this;
-      function createButtonHelper(card) {
+      function createCardElemHelper(card) {
         if(card instanceof Folder || card instanceof Deck) {
           return;
         }
 
-        var appButton = CardUtil.createCardButton(card);
-        that.gridView.appendChild(appButton);
-        that.appButtons.push(appButton);
-        return appButton;
+        var nodeElem = document.createElement('div');
+        nodeElem.appendChild(CardUtil.createCardFragment(card));
+        that.gridView.appendChild(nodeElem);
+        return nodeElem;
       }
 
       folderList && folderList.forEach(card => {
-        var appButton = createButtonHelper(card);
-        if (appButton) {
-          appButton.dataset.parentType = 'folder';
-          appButton.classList.add('selected');
+        var appCardElem = createCardElemHelper(card);
+        if (appCardElem) {
+          var cardButton = appCardElem.firstElementChild;
+          cardButton.dataset.parentType = 'folder';
```

Comment:
For now `parentType` is used only in `updateFolder` to determine if we should move it out of folder (if it was a folder's card), or put it into a folder (if it was a main list's card), or do nothing. This is not used elsewhere since there's no other usecases that need it.